### PR TITLE
[Docs] Add default cloud core settings

### PR DESCRIPTION
**Pull request by justindeguzman**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Add default cloud core settings

### CODE REVIEW

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit fe6d8316a35ce5ca93759a3eac2e36193476af22 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/59783/fe6d8316a35ce5ca93759a3eac2e36193476af22/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/59783/fe6d8316a35ce5ca93759a3eac2e36193476af22/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -1126,7 +1128,9 @@ Possible values:
 - 0 (or 1) — `INSERT SELECT` no parallel execution.
 - Positive integer. Bigger than 1.
 
-Default value: 0.
+Default value: `0`.
+
+Cloud default value: `2`.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/settings.md:1133]:
```suggestion
Cloud default value: from `2` to `4`, depending on the service size.
```

File: docs/en/operations/settings/settings.md [hunk 2]
```diff
@@ -2132,6 +2140,8 @@ Possible values:
 
 Default value: 0
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/settings.md:2141]:
```suggestion
Default value: 20
```

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -2660,6 +2670,8 @@ Type: [UInt64](../../sql-reference/data-types/int-uint.md).
 
 Default value: 1000000000 nanoseconds (once a second).
 
+Cloud default value: `0`.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/settings.md:2673]: It is temporarily disabled.

File: docs/en/operations/settings/settings.md [hunk 4]
```diff
@@ -2683,6 +2695,8 @@ Type: [UInt64](../../sql-reference/data-types/int-uint.md).
 
 Default value: 1000000000 nanoseconds.
 
+Cloud default value: `0`.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/settings.md:2698]: It is temporarily disabled.

File: docs/en/operations/settings/settings.md [hunk 5]
```diff
@@ -3319,7 +3335,9 @@ Possible values:
 
 - a string representing any valid table engine name
 
-Default value: `None`
+Default value: `None`.
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/settings.md:3338]:
```suggestion
Default value: `MergeTree`.
```

**Pull request merged**